src/Microsoft.AspNet.Razor/CodeGenerators/CSharpCodeWriter.cs

```diff
@@ -470,23 +470,6 @@ private void WriteCStyleStringLiteral(string literal)
                         Write(literal[i].ToString());
                         break;
                 }
-                if (i > 0 && i % 80 == 0)
-                {
-                    // If current character is a high surrogate and the following
-                    // character is a low surrogate, don't break them.
-                    // Otherwise when we write the string to a file, we might lose
-                    // the characters.
-                    if (Char.IsHighSurrogate(literal[i])
-                        && (i < literal.Length - 1)
-                        && Char.IsLowSurrogate(literal[i + 1]))
-                    {
-                        Write(literal[++i].ToString());
-                    }
-
-                    Write("\" +");
-                    Write(NewLine);
-                    Write("\"");
-                }
```

Comment:
Probably fine. Just worth mentioning separately in the squished (or squashed) commit description.